packages/react-markdown/src/primitives/MarkdownText.tsx

```
@@ -58,76 +59,89 @@ export type MarkdownTextPrimitiveProps = Omit<
+    const status = useSmoothStatus();
     return (
       <Container
         data-status={status.type}
         {...containerProps}
         className={classNames(className, containerProps?.className)}
         ref={forwardedRef}
       >
-        <ReactMarkdown components={components} {...rest}>
-          {text}
-        </ReactMarkdown>
+        <MarkdownTextInner {...rest}></MarkdownTextInner>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)